tests/generators/schema/author-test.php

```diff
@@ -82,33 +122,16 @@ public function setUp() {
 	/**
 	 * Tests that the author gets a 'mainEntityOfPage' property pointing to the webpage Schema piece on the same page.
 	 *
-	 * @covers Yoast\WP\SEO\Presentations\Generators\Schema\Author::generate
+	 * @covers ::generate
+	 * @covers ::determine_user_id
 	 */
-	public function test_generate() {
+	public function test_generate_on_author_pages() {
 		$user_id = 123;
-		$person_data = [
-			'@type' => [
-				'Person',
-			],
-      		'@id'   => 'http://basic.wordpress.test/#/schema/person/a00dc884baa6bd52ebacc06cfd5aab21',
-      		'name'  => 'Ad Minnie',
-      		'image' => [
-				'@type' => 'ImageObject',
-        		'@id'   => 'http://basic.wordpress.test/#personlogo',
-        		'url'   => 'http://2.gravatar.com/avatar/e64c7d89f26bd1972efa854d13d7dd61?s=96&d=mm&r=g',
-        		'caption' => 'Ad Minnie',
-      		],
-      		'sameAs' => [
-				'https://facebook.example.org/admin',
-				'https://instagram.example.org/admin',
-				'https://linkedin.example.org/admin',
-			],
-		];
 
 		$this->instance->expects( 'build_person_data' )
 					   ->once()
 					   ->with( $user_id, $this->meta_tags_context )
-			           ->andReturn( $person_data );
+			           ->andReturn( $this->person_data );
```

Comment:
The tabs and spaces should not be mixed. This happens in more, please adapt in other places too.
You can change the PHPStorm settings: Editor --> Code Style --> PHP --> Wrapping and Braces -> Chained methods calls to `Chop down if long` and uncheck `Align when multiline`. Then the auto-format will do this for you correctly.

Reply:
I fixed the code style settings in PHPStorm and fixed the indentation with "the claw" 🙂.